nereid/nereid/data/default_data/README.md

```diff
@@ -0,0 +1,87 @@
+# How to data
+
+Welcome to the documentation about reference data! If you are reading this, it's because you need to be able to load project specific data into this tool. That is great, and it is fully supported by the both the development build pattern, and the production build pattern (forthcoming). 
```

Comment:
each sentence in markdown should be on a new line. makes it easier to read and edit